grafana-plugin/src/models/user/user.ts

```diff
@@ -143,6 +143,14 @@ export class UserStore {
     this.loadCurrentUser();
   }
 
+  async disconnectGoogle() {
+    // TODO: update this to use the openAPI autogenerated schema stuff
+    // @ts-ignore TODO: remove this
```

Comment:
@maskin25 can these comments be removed?

Reply:
yes, it should be removed, the problem that typescript consider the line below as error and `yarn generate-types` doesn't work for me locally, could you generate the types and push the changes?

Reply:
what errors do you get w/ `yarn generate-types`?

Reply:
I fixed the issue with `yarn generate-types` and regenerated the types however there no changes in the `grafana-plugin/src/network/oncall-api/autogenerated-api.types.d.ts` happened and ts highlights url
 await onCallApi().GET(**'/disconnect/google-oauth2/**', undefined);  with red, did you update OpenAPI schema here?

Reply:
in fact, `autogenerated-api.types.d.ts` is up to date (just double checked, there's no new output)

Reply:
hmm, any ideas why ts considers the following urls
```
 onCallApi().GET('/login/google-oauth2/', undefined);
and
onCallApi().GET('/disconnect/google-oauth2/', undefined);
```
non existing

Reply:
what're the typescript errors that you see if you remove `// @ts-ignore`?

Reply:
I'm not very familiar w/ how the autogenerated API client is to be consumed on the frontend but I presume you should do something similar to [this](https://github.com/grafana/oncall/blob/dev/grafana-plugin/src/models/user/user.ts#L148-L149)?:
```ts
await onCallApi().PUT('/users/{id}/', {
        params: { path: { id: data.pk } },
```

you probably can't hardcode `google-oauth2`, it should _probably_ be `{backend}` and then maybe add `params` and populate that variable?

Reply:
yes, you are right, thank you! I fixed and pushed